bodhi/client/__init__.py

```diff
@@ -370,11 +370,25 @@ def edit(user, password, url, **kwargs):
             resp = client.query(**query_param)
             title = resp['updates'][0]['title']
         elif re.search(bindings.UPDATE_TITLE_RE, kwargs['update']):
+            query_param = {'like': kwargs['update']}
+            resp = client.query(**query_param)
             title = kwargs['update']
         del(kwargs['update'])
         kwargs['builds'] = title
         kwargs['edited'] = title
 
+        data = munch.unmunchify(resp)
+        if 'updates' in data:
+            former_update = data['updates'][0]
+        else:
+            former_update = data
+
+        # Replace None params (not provided) with previous database values
+        for k in kwargs:
+            if not kwargs[k] and k in former_update:
```

Comment:
What happens here when `kwargs[k] is False` ? it will use the value of `former_update` which is not what we expect.
I think here checking for `kwargs[k] is None and ...` would work.

Reply:
Yes, that is a good point. Will make the changes.

Although, do we modify in case of empty lists ```[]``` and empty strings ```""```?